products/memcache/api.yaml

```diff
@@ -0,0 +1,147 @@
+# Copyright 2018 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: Memcache
+versions:
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://memcache.googleapis.com/v1beta2/
+scopes:
+  - https://www.googleapis.com/auth/cloud-platform
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+    resource_inside_response: true
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: True
+    allowed:
+      - True
+      - False
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+objects:
+  - !ruby/object:Api::Resource
+    name: 'Instance'
+    min_version: beta
+    create_url:  projects/{{project}}/locations/{{region}}/instances?instanceId={{name}}
+    self_link: projects/{{project}}/locations/{{region}}/instances/{{name}}
+    base_url:  projects/{{project}}/locations/{{region}}/instances
+    update_verb: :PATCH
```

Comment:
Can you add a handwritten update test for this resource? Also an import test because this is `beta` only, it doesn't generate one